README.md

```diff
@@ -131,14 +131,3 @@ defmodule MyModule.SampleTest do
   end
 end
 ```
-
```

Comment:
I think that user is able to see for what versions of Elixir/Erlang the library works by checking `.github/workflows/main.yml` and `mix.exs`.

I kept `elixir: "~> 1.12",` in [`mix.exs`](https://github.com/wise-home/wise_homex/blob/master/mix.exs#L8C6-L8C25) assuming that library was used since its beginning with all versions => 1.12. I am not sure if this is the right approach as CI is testing builds since Elixir 1.13. Maybe we should keep CI builds for Elixir 1.12? 🤔 

I was basing on Jason Elixir library:
- https://github.com/michalmuskala/jason/blob/master/.github/workflows/ci.yml#L16
- https://github.com/michalmuskala/jason/blob/master/mix.exs#L11

Reply:
I'm okay with removing this section. I don't believe it's necessary to keep the CI builds for 1.12.